style.css

```
@@ -80,3 +80,245 @@ ul {
 .fa-solid {
   color: #fff;
 }
+
+/* About myself section */
+.About-myself {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 20px;
+}
+
+.Recent-work {
+  grid-column: 1/3;
+}
+
+.Recent {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #1c1a19;
+  height: 83px;
+}
+
+.line {
+  border: 1px solid #212121;
+}
+
+.about-img {
+  height: 237px;
+  margin-top: 50px;
+}
+
+.Multi-Post-Heading {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.Multi-Post-stories {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #091e42;
+  margin-top: 12px;
+  line-height: 24px;
+}
+
+.Multi-Post-ul {
+  width: 100%;
+  margin-bottom: 15px;
+}
+
+.Multi-Post-li {
+  border: 1px solid #8993a4;
+  padding: 9px 12px;
+  margin-right: 10px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+}
+
+.Multi-Post-project {
+  align-items: center;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  border-style: none;
+}
+
+/* work section */
+.work-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 20px;
+}
```

- [ ] As mentioned by the previous reviewer kindly set the column to just 1 for a good design and user experience please check below :point_down:  my suggestions:

```suggestion
.work-container {
  display: grid;
  grid-template-columns: repeat(1, 1fr);
  grid-gap: 20px;
}
```